docs/source/support/component-statuses.rst

```diff
@@ -124,18 +124,24 @@ Libraries
    * - ``bindings-rxjava`` :doc:`library and its public API </app-dev/bindings-java/index>`.
      - Stable
      -
-   * - Maven artifact ``daml-lf-1.6-archive-java-proto``
+   * -  ``daml-lf-1.6-archive-java-proto``
      - Stable
      -
-   * - Maven artifact ``daml-lf-1.7-archive-java-proto``
+   * -  ``daml-lf-1.7-archive-java-proto``
      - Stable
      -
-   * - Maven artifact ``daml-lf-1.8-archive-java-proto``
+   * -  ``daml-lf-1.8-archive-java-proto``
      - Stable
      -
-   * - Maven artifact ``daml-lf-dev-archive-java-proto``
+   * -  ``daml-lf-dev-archive-java-proto``
      - Alpha
      -
+   * - **Python Ledger API Bindings**
+     -
+     -
+   * - ``dazl`` `library and its public API <https://digital-asset.github.io/dazl-client/>`_
```

Comment:
Didn't we agree the dazl name was deprecated and shouldn't be use outside of "formerly known as"?

Reply:
Agreed, however that's referencing the actual library which won't change its name, Davin said that it's too complicated so that will have to remain.

Reply:
@da-tanabe What are the obstacles to renaming the library? Is it complete blocked or depends on creating a new major version with breaking changes?